dev/com.ibm.ws.jsfContainer_fat_2.3/fat/src/com/ibm/ws/jsf/container/fat/tests/JSFContainerTest.java

```diff
@@ -52,7 +52,10 @@ public static void setUp() throws Exception {
 
     @AfterClass
     public static void tearDown() throws Exception {
+      // Stop the server
+      if (server != null && server.isStarted()) {
         server.stopServer();
+      }
```

Comment:
Saw these in other tests, so I thought it should be there too?

Reply:
The `isStarted()` isn't completely necessary since `stopServer()` checks that, but it doesn't hurt

Reply:
I'll just keep it for consistency